extern/recastnavigation/DebugUtils/Include/DebugDraw.h

```diff
@@ -206,11 +206,11 @@ class duDisplayList : public duDebugDraw
 public:
 	duDisplayList(int cap = 512);
 	~duDisplayList();
-	virtual void depthMask(bool state);
-	virtual void begin(duDebugDrawPrimitives prim, float size = 1.0f);
-	virtual void vertex(const float x, const float y, const float z, unsigned int color);
-	virtual void vertex(const float* pos, unsigned int color);
-	virtual void end();
+    virtual void depthMask(bool state) override;
+    virtual void begin(duDebugDrawPrimitives prim, float size = 1.0f) override;
+    virtual void vertex(const float x, const float y, const float z, unsigned int color) override;
+    virtual void vertex(const float* pos, unsigned int color) override;
+    virtual void end() override;
```

Comment:
Since you set recastnavigation to use the `no-suggest-override` flag, then we can revert these right?